x-pack/plugins/endpoint/common/types.ts

```diff
@@ -5,6 +5,11 @@
  */
 
 import { SearchResponse } from 'elasticsearch';
+import { schema, TypeOf } from '@kbn/config-schema';
```

Comment:
Not about your code changes but should we break up the `types.ts` file into smaller files specific to an API like:
A file for `alerts`, `resolver`, `metadata` and then have an `index.ts` file that puts them altogether so this `types.ts` file doesn't get too large?